util/src/main/proto/Mcf.proto

```diff
@@ -31,9 +32,14 @@ message McfGraph {
     repeated TypedValue typed_values = 1;
   }
 
-  // Map of a property name to its values.
+  // Information about a node.
   message PropertyValues {
+    // Map of a property name to its values.
     map<string, Values> pvs = 1;
+
+    // Information identifying the location of this node in the source.
+    // There can be multiple if PVs in this node are merged from different files.
+    repeated Log.Location location = 2;
```

Comment:
should Location be outside Log proto? It seems like a general thing.

Reply:
Location sounded like too general a name :). At the moment its only planned for use in the context of logging, debugging, msgs... do you have an alternate name suggestion?